internal/controller/cluster/reconciler.go

```diff
@@ -319,7 +321,9 @@ func (r *SingleClusterReconciler) recoverIgnorablePods() common.ReconcileResult
 		return common.ReconcileRequeueAfter(0)
 	}
 
-	return common.ReconcileSuccess()
+	r.Log.Info("Found ignorable pod(s), requeuing")
+
+	return common.ReconcileRequeueAfter(0)
```

Comment:
The return statement is the same in both lines 321 and 326, so it doesn't need to be duplicated.
If the goal is to differentiate log messages based on conditions, you can handle that separately before a single return.
Or you can place the log message as soon as you get the failed pod under `CheckPodFailed` condition.